src/reversed_sequence.py

```diff
@@ -0,0 +1,13 @@
+def reverse_sequence(n):
+    solution = []
+    subtractor = 1
+    for i in range(n):
+        if i == 0:
+            solution.append(n)
+        else:
+            solution.append(n - subtractor)
+            subtractor += 1
+    return solution
```

Comment:
food for though:
```python
def reverse_sequence(n):
    return [m for m in range(n, -1, -1)]
```
Range takes a start, stop, and a step. https://docs.python.org/3/library/functions.html#func-range